control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -433,8 +605,9 @@ class Pid
   double p_error_last_; /** Save state for derivative state calculation. */
   double p_error_;      /** Error. */
   double d_error_;      /** Derivative of error. */
-  double i_term_;       /** Integrator state. */
+  double i_term_{0};    /** Integrator state. */
   double cmd_;          /** Command to send. */
+  double cmd_unsat_;    /** command without saturation. */
```

Comment:
```suggestion
  double i_term_ = 0.0;    /** Integrator state. */
   double cmd_ = 0.0;          /** Command to send. */
  double cmd_unsat_ = 0.0;    /** command without saturation. */
```
Let's initialize all the member variables

Reply:
Fixed in commit e1b7413.